pkgs/top-level/python-packages.nix

```
@@ -2635,7 +2635,7 @@ in {
 
   paperwork-backend = callPackage ../applications/office/paperwork/backend.nix { };
 
-  papis = callPackage ../development/python-modules/papis { };
+  papis = disabledIf (!isPy3k) (callPackage ../development/python-modules/papis { });
```

please move this into the expression
```suggestion
  papis =callPackage ../development/python-modules/papis { };
```